metacat-common-server/build.gradle

```diff
@@ -40,7 +40,8 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter")
     compile("org.springframework.boot:spring-boot-starter-logging")
     compile("org.springframework.cloud:spring-cloud-starter-spectator")
-
+    compile("org.springframework:spring-jdbc")
```

Comment:
you mean 'spring-boot-starter-jdbc' , which seems not including the jdbc-template classes, and I have to explicitly depend on spring-jdbc